src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -38,13 +38,27 @@ export class ClaimAmountRow {
     return new ClaimAmountRow(undefined, undefined)
   }
 
+  static translateToNumber (input?: any): any {
+    const numberValue = _.toNumber(input)
+    const isExponential = numberValue.toString().match(new RegExp('\\d*[Ee][+-]?\\d*?'))
+
+    if (!isNaN(numberValue) && !isExponential) {
+      return numberValue
+    } else {
+      return input
+    }
+  }
+
   static fromObject (value?: any): ClaimAmountRow {
     if (!value) {
       return value
     }
 
     const reason = value.reason || undefined
-    const amount = value.amount ? _.toNumber(value.amount) : undefined
+    const amount = value.amount
+      ? ClaimAmountRow.translateToNumber(value.amount.replace(new RegExp(/,/g), ''))
```

Comment:
I think it's not necessary to do `new RegExp`, `/,/` literal should be enough.

Reply:
@dharmendrak I do not see a reason why removing coma is done in one place and while conversion to numeric type is done in another place. Would you expect parsing function in Java to accept pre-processed value? Please keep both things together, the goal if I understand correctly is to handle numbers with coma in deserialisation.

Reply:
@geek-soft Can interest rate be entered more than three digits.
@rlewan this replace remove all , in the number, I mean it can be more than one.